docs/core/tools/dotnet-new.md

```diff
@@ -508,18 +738,26 @@ Create a .NET Standard class library project in the specified directory (availab
 
 `dotnet new classlib -lang VB -o MyLibrary`
 
-Create a new ASP.NET Core C# MVC application project in the current directory with no authentication:
+Create a new ASP.NET Core C# MVC project in the current directory with no authentication:
 
 `dotnet new mvc -au None`
 
-Create a new xUnit application:
+Create a new xUnit project:
 
 `dotnet new xunit`
 
 List all templates available for MVC:
 
 `dotnet new mvc -l`
 
+List all templates matching the *we* substring. No exact match is found, so substring matching runs against both the short name and name columns.
```

Comment:
do we need to since when this is supported?

Reply:
@seancpeters Has the template substring matching in `dotnet new` been supported since the beginning?

Reply:
@scottaddie - Yes, it has been supported from very early on.

Reply:
Since the command `dotnet new we` doesn't have the `-l` flag, this command is more about creating a template than listing templates. Listing matches is a fallback behavior, which occurs when it can't be decided which template to invoke.